src/plus/startWork/startWork.ts

```diff
@@ -570,3 +594,30 @@ function isStartWorkTypeItem(item: unknown): item is StartWorkTypeItem {
 export function getStartWorkItemIdHash(item: StartWorkItem) {
 	return md5(item.item.issue.id);
 }
+
+function buildItemActionTelemetryData(item: StartWorkItem) {
```

Comment:
Nitpick: this is used outside of "action" contexts, and the `action` property is really only being set in `sendItemActionTelemetry`, so maybe we just call this `buildItemTelemetryData`?

Reply:
fixed